Code/skyrim_ui/src/app/components/dropdown/dropdown.component.html

```diff
@@ -7,8 +7,7 @@
   <div class="box" (click)="toggle($event)">
     <ng-container>
       <div class="dropdown-selected">
-        <span>{{ options?.[selected] ? options[selected].text : placeholder }}</span>
-        <fa-icon [icon]="faCaretDown"></fa-icon>
+        <span>{{ noTranslate ? getSelectedLabel() : (getSelectedLabel() | transloco: {length: options[selected]?.value}) }}</span>
```

Comment:
```suggestion
        <span>{{ noTranslate ? selectedLabel : (selectedLabel | transloco: { length: options[selected]?.value }) }}</span>
```
Could you convert the function to a getter. Using functions in template is considered non optimal practice